roles/reset/tasks/main.yml

```diff
@@ -54,3 +54,31 @@
   file:
     path: /tmp/k3s
     state: absent
+
+- name: Check if rc.local exists
+  stat:
+    path: /etc/rc.local
+  register: rcfile
```

Comment:
Can you explain how the LXC specific tasks in the `reset` role are different from the tasks in the `reset_proxmox_lxc` role?

Reply:
reset tasks run on the actual container running k3s (the lxc containers) reset_proxmox_lxc runs on the proxmox hosts directly.

Reply:
I see 👍🏻